### PR TITLE
Initial assimilation of 0.7.0 to Rally

### PR DESCRIPTION
**Pull request by jeffmay**
@patrick-bohan @usufruct99 @mingy711 @audax-shreyaspurohit 

This library allows better integration with MacWire and compile-time constructor injection.

### CODE REVIEW

**Comment by audax-shreyaspurohit** [comment]
lgtm

**Comment by jeffmay** [comment]
retest this please

File: LICENSE [hunk 1]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright {yyyy} {name of copyright owner}
+   Copyright 2015 Jeff May
```
Comment (blin86) [inline LICENSE]: wut? You own this?

**Comment by blin86** [comment]
lGTM

File: LICENSE (hunk 1 above)
Comment (jeffmay) [inline LICENSE]: This repo is open-source under the Apache 2 license.

**Pull request merged**